src/components/JarPage/JarProgress.jsx

```diff
@@ -0,0 +1,71 @@
+import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress';
+import { styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { useDispatch, useSelector } from 'react-redux';
+import Grid from '@mui/material/Grid';
+
+const BorderLinearProgress = styled(LinearProgress)(({ theme }) => ({
+    height: 40,
+    borderRadius: 20,
+    [`&.${linearProgressClasses.colorPrimary}`]: {
+      backgroundColor:
+        theme.palette.grey[theme.palette.mode === 'light' ? 200 : 800],
+    },
+    [`& .${linearProgressClasses.bar}`]: {
+      borderRadius: 20,
+      background:
+        'linear-gradient(270.27deg, #58D68D 1.94%, rgba(88, 214, 141, 0.51) 99.95%);',
+    },
+  }));
```

Comment:
We have a reusable component SumLinearProgress in components folder, maybe you could use it here?